go-controller/pkg/ovn/base_network_controller_policy.go

```diff
@@ -58,6 +58,10 @@ type defaultDenyPortGroups struct {
 	policies map[string]bool
 }
 
+func (bnc *BaseNetworkController) getPortGroupName(readableGroupName string) string {
```

Comment:
I don't if my comment is getting lost, I will comment here again
The point was to do both hashing and prefixing for port group name in one function like (the order of hashing and prefixing may be different)

func (ncni *NetworkControllerNetInfo) getPortGroupName(name string) string {
	networkScopedName := ncni.GetNetworkScopedName(name)
	return hashedPortGroup(networkScopedName)
}

that is to remove the logic of hashing/prefixing form the callers. 
We don't hash ClusterPortGroupName and ClusterRtrPortGroupName, and I think this function is a good place to unify this strange logic too, so it can skip hashing for these port groups in one place

For the same reason I suggest to pass only name to buildPortGroup, and then buildPortGroup itself may get the name of the port group by calling getPortGroupName
Please let me know if that makes sense

Reply:
talked to @npinaeva offline. It is hard to unify like she suggested as the defaultDenyPortGroup/policyPortGroup/multicastPortGoup all use different form. agreed to defer this until we agreed on a solution.

Reply:
I guess at this point this function is not needed, we will have a separate function for every type of port group to get its name?